### PR TITLE
Applied latest markup to post template.

### PR DESCRIPTION
**Pull request by kirdia**
Changed single post template according to latest markup.
Changed comment form elements according to latest markup.
Top background image is disabled for the moment.

### CODE REVIEW

File: templates/comments_section.twig [hunk 1]
```diff
@@ -1,16 +1,13 @@
-<div class="container">
+<div class="container page-section">
+	<h2 class="page-section-header">{{ __( 'Comments', domain ) }}</h2>
```
Comment (sagarsdeshmukh) [inline templates/comments_section.twig]: I guess ` __( 'Comments', domain )` - here we need to use `__( 'Comments', 'planet4-master-theme' )` ?
Please confirm @koyan @angtheod

Comment (koyan) [reply templates/comments_section.twig]: Comments exists as a localised message in wordpress, so we don't need to retranslate it.

Comment (koyan) [reply templates/comments_section.twig]: But I am not sure if we get that by using "domain" there. Have you confirmed that @kirdia ?

Comment (kirdia) [reply templates/comments_section.twig]: No, I haven't checked if the text domain was working properly, but you are right. We do not need to retranslate common words like 'Comments', 'Comment' etc. I will remove the planet4 master theme text domain for those.

**Pull request merged**